packages/anac-certified-attributes-importer/test/download-csv.test.ts

```diff
@@ -0,0 +1,128 @@
+import sftp from "ssh2-sftp-client";
+import { describe, expect, it } from "vitest";
+import { SftpClient } from "../src/service/sftpService.js";
+import { sftpConfigTest } from "./helpers.js";
+
+describe("CSV getFileName", () => {
+  const fileInfoMock: sftp.FileInfo = {
+    type: "-",
+    name: "",
+    size: 0,
+    modifyTime: 0,
+    accessTime: 0,
+    rights: {
+      user: "user",
+      group: "group",
+      other: "other",
+    },
+    owner: 0,
+    group: 0,
+  };
```

Comment:
I see lots `name` property overrides in the tests, to avoid spread operator could be useful to create a function with optional parameter `name` that return a mocked file info:
```suggestion
  const getMockFileInfo = (name? : string): sftp.FileInfo => ({
    type: "-",
    name: name || "",
    size: 0,
    modifyTime: 0,
    accessTime: 0,
    rights: {
      user: "user",
      group: "group",
      other: "other",
    },
    owner: 0,
    group: 0,
  };
)};
```